sdk/textanalytics/Azure.AI.TextAnalytics/tests/SessionRecords/AnalyzeSentimentTests/AnalyzeSentimentBatchConvenienceFullTest.json

```diff
@@ -135,8 +135,6 @@
     }
   ],
   "Variables": {
-    "RandomSeed": "508530765",
-    "TEXT_ANALYTICS_API_KEY": "Sanitized",
-    "TEXT_ANALYTICS_ENDPOINT": "https://cognitiveusw2dev.azure-api.net"
```

Comment:
last time I did this, CI failed for me as it needed this variables. curious into why they got deleted

Reply:
updated the env variables.